products/sle/main.pm

```diff
@@ -703,6 +703,7 @@ elsif (get_var("SUPPORT_SERVER")) {
         load_inst_tests();
     }
     loadtest "ha/barrier_init" if get_var("HA_CLUSTER");
+    loadtest "hpc/control"     if get_var("HPC");
```

Comment:
let's call it in a same way as HA - barrier_init

Reply:
renamed.. Maybe I was too much creative with renaming  supportserver to control, or is it fine?